lib/bamboo/test.ex

```diff
@@ -82,31 +82,28 @@ defmodule Bamboo.Test do
   checked when using the assertion helpers like `assert_delivered_email/1`.
 
   Sometimes emails don't show up when asserting because you may deliver an email
-  from a _different_ process than the test process. When that happens, set the
-  `process_name`. This will name the test process using `Process.register/2`
-  and set `Bamboo.TestAdapter` to always send to that process. This means
-  that you cannot use `process_name` with async tests.
+  from a _different_ process than the test process. When that happens, turn on
+  shared mode. This will tell `Bamboo.TestAdapter` to always send to the test process.
+  This means that you cannot use shared mode with async tests.
 
   ## Try to use this version first
 
       use Bamboo.Test
 
-  ## And if you are delivering from another process, set `process_name`
-
-      # Note: the process name can be whatever you want.
-      use Bamboo.Test, process_name: :name_of_my_test
+  ## And if you are delivering from another process, set `shared: true`
 
+      use Bamboo.Test, shared: true
 
   Common scenarios for delivering mail from a different process are when you
   send from inside of a Task, GenServer, or are running acceptance tests with a
   headless browser like phantomjs.
   """
-  defmacro __using__(process_name: process_name) do
+  defmacro __using__(shared: true) do
     quote do
       setup tags do
         if tags[:async] do
           raise """
-          passing process_name to Bamboo.Test cannot be done for async tests.
+          using shared mode with Bamboo.Test cannot be done for async tests.
```

Comment:
I like this, but missed it during your review! I’ll add it in later
> On Apr 26, 2016, at 9:41 AM, Josh Steiner <notifications@github.com> wrote:
> 
> In lib/bamboo/test.ex <https://github.com/thoughtbot/bamboo/pull/136#discussion_r61086620>:
> 
> >      quote do
> >        setup tags do
> >          if tags[:async] do
> >            raise """
> > -          passing process_name to Bamboo.Test cannot be done for async tests.
> > +          using shared mode with Bamboo.Test cannot be done for async tests.
> Also, thoughts on changing this from the passive voice <https://en.wikipedia.org/wiki/English_passive_voice> to make it a little more clear? Maybe something like You cannot use Bamboo.Test shared mode with async tests?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/136/files/395f355edeb979b36004315678ee782dad446b46#r61086620>